apps/server/src/modules/instance-config/entity/instance-config.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { BaseEntityWithTimestamps } from '@shared/domain/entity/base.entity';
+import { EntityId } from '@shared/domain/types';
+
+export interface InstanceConfigEntityProps {
+	id?: EntityId;
+
+	name: string;
+}
+
+@Entity({ tableName: 'instance-configs' })
```

Comment:
```suggestion
@Entity({ tableName: 'instance' })
```

Reply:
called it "instances" to fit the collections name schema